app/admin/pup.rb

```diff
@@ -8,6 +8,18 @@
   filter :breed
   filter :pup_name
   actions :all, except: [:update, :new, :edit]
+  batch_action :destroy, confirm: "Are you sure you want to delete these pups?" do |ids|
```

Comment:
I see what you're trying to do here. But maybe we shouldn't be copy/pasting code. Consider refactoring a little bit?